configure.ac

```diff
@@ -32,18 +32,28 @@ MONO_VERSION_MAJOR=`echo $VERSION | cut -d . -f 1`
 MONO_VERSION_MINOR=`echo $VERSION | cut -d . -f 2`
 MONO_VERSION_BUILD=`echo $VERSION | cut -d . -f 3`
 
-# This is the version number of the corlib-runtime interface. When
+#
+# This is the version of the corlib-runtime interface. When
 # making changes to this interface (by changing the layout
 # of classes the runtime knows about, changing icall signature or
-# semantics etc), increment this variable.
+# semantics etc), change this variable.
+#
+# This must be unique relative to corlib interface/semantics.
+#
+# Use uuidgen or make up a really reliably unique string, such as a git hash
+# or github PR number. Do not use anything personally identifiable or not particularly unique --
+# names, birthdays, branch names -- or use funny characters that
+# will not interoperate with C, C++, sh, make, etc. -- spaces, percents, stars, dollars, parens, etc.
```

Comment:
I'd remove all the trivia about PII/birthdays etc. and just mention the uuidgen command you need to generate the string in the right format.